components/docs-chef-io/content/automate/backup.md

```diff
@@ -31,7 +29,7 @@ This amount of space needed for a backup varies depending on your Chef Automate
 * Elasticsearch snapshots of your Chef Automate configuration and data, such as converge, scan, and report data. You will need enough disk space for the each Elasticsearch snapshot and the delta--or the list of changes--for each successive snapshot
 * Chef Habitat Builder artifacts
 
-## Set Up Backups to a Filesystem
+## Backup to a Filesystem
```

Comment:
@mjingle The issue I have with "Backup" vs "Set up Backups..."  as a header for this section is that we aren't actually telling people how to create the backup in the section, only how to set things up so that when they do create a backup, it will work.

Reply:
@yzl In reading the content, isn't the configuration part implied? Also, in reading the page's table of contents in the righthand side as viewed via the netlify link, users can locate the actual "Backup Commands" > "Create Backup" section.

Reply:
@mjingle Those sections are a combination of (1) how to configure Automate to put your backup where you want it, and (2) how to set up the surrounding systems so that your backup lands where it should (e.g. how to put the right permissions on your s3 buckets and what not, where shared filesystems need to be mounted, and so forth). It's definitely a lot, but it's all a precursor to being able to run `chef-automate backup create` succesfully.